src/manager.js

```diff
@@ -26,10 +26,11 @@ let roomData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/rooms/r
   .then(data => data.rooms)
 
 Promise.all([bookingData, roomData]).then((requiredData) => {
-  const rooms = requiredData[1];
-  const booked = requiredData[0];
-  let bookedTodayData = manager.booked(booked);
-  $('.revenue').html(manager.totalSpend(rooms, bookedTodayData).toFixed(2));
-  $('.percentage').html((bookedTodayData.length/rooms.length) * 100)
-  manager.availableToday(rooms, bookedTodayData).forEach(room => {$('.available').append(`<span class="rooms">${room}</span>`)})
+  manager.rooms = requiredData[1];
+  manager.booked = requiredData[0];
```

Comment:
I would consider instantiating your manager class here and passing these through as parameters as well.